content/docs/enterprise/changelog.mdx

```diff
@@ -15,6 +15,34 @@ import Versioning from '../admonitions/_semantic-versioning.mdx';
 
 :::
 
+## 0.27.0
+
+### Improvements
```

Comment:
I think the other versions use the label "New" for section.

Reply:
Updated.